mne/simulation/source.py

```diff
@@ -4,13 +4,99 @@
 #
 # License: BSD (3-clause)
 
+import os.path as op
+
 import numpy as np
 
+from ..label import grow_labels, Label
 from ..source_estimate import SourceEstimate, VolSourceEstimate
 from ..source_space import _ensure_src
+from ..surface import read_surface
 from ..utils import check_random_state, warn, _check_option
 
 
+def select_sources(subject, label, location='center', extent=0,
+                   grow_outside=True, subjects_dir=None, name=None,
+                   surf='white'):
+    """Select the sources used for simulation
```

Comment:
This is actually more general than sim so I would write

Select a subset of sources from a label

Also I'd move it to `mne/label.py` for the same reason, then import into the `mne/__init__.py` namespace and add to a suitable location in `python_reference.rst`, probably wherever `grow_labels` is (Travis will probably complain about this if it ever manages to finish)

Reply:
Made the change, but used `Select sources from a label.` because the source are not necessarily a subset because of the grow_outside option.